frontend/src/clusterView/clusterView.js

```diff
@@ -54,9 +54,13 @@ export class ClusterView extends React.Component {
 
     render() {
         if (this.state.loading) {
-            return (<h1>
-                Loading!
-            </h1>);
+            return (
+                <React.Fragment>
```

Comment:
This pattern is in a bunch of places as a loading screen component -- let's factor that out.